### ISSUE TITLE
New site?

### ISSUE THREAD

**Issue by Skankhunter42** [spaam/svtplay-dl#463]
Hello.

Is it possible to add a site behind a pay wall to the list of supported sites? The site I'm reffering to is www.sli.se 

Thanks

**Comment by spaam** [comment]
yes.  we have support for similar things right now (tv4play premium and dplay premium content).

**Comment by Skankhunter42** [comment]
Is it possible to enable support for that perticular site www.sli.se

**Comment by spaam** [comment]
i dont have access to the site, someone with access need to fix it.

**Comment by Skankhunter42** [comment]
So if I gave you access can you fix it then?

**Comment by Skankhunter42** [comment]
It's above what I can do with my coding skills.

**Comment by spaam** [comment]
sure you can give me access to it, you can user / passwd to my mail j@i19.se
